pom.xml

```diff
@@ -15,7 +15,7 @@
     <groupId>org.mule.connectors</groupId>
     <artifactId>mule-http-connector</artifactId>
     <packaging>mule-extension</packaging>
-    <version>1.8.0-SNAPSHOT</version>
+    <version>1.8.1-SNAPSHOT</version>
```

Comment:
We change the snapshot version once we release a connector. Let's change it again to 1.8.0

Reply:
Done